service-names.py

```diff
@@ -0,0 +1,11 @@
+import yaml
+
+with open('docker-compose-template.yaml', 'r') as f:
+    data = yaml.load(f, Loader=yaml.SafeLoader)
+
+# print(type(data["services"].keys()))
+
+with open('service-names.txt', 'w') as f:
+    services = list(data["services"].keys())
+    for service in services:
+        f.write(service + '\n')
```

Comment:
Ideal approach in my opinion, this handles all services regardless of their naming convention in a general way. Good use of the python context handler for working with files. Small nitpick, the print statement used for debugging purposes should be removed.